internal/repository/index_parallel.go

```diff
@@ -0,0 +1,77 @@
+package repository
+
+import (
+	"context"
+	"sync"
+
+	"github.com/restic/restic/internal/debug"
+	"github.com/restic/restic/internal/restic"
+	"golang.org/x/sync/errgroup"
+)
+
+const streamIndexParallelism = 5
```

Comment:
Maybe `loadIndexParallelism` to be conform with the other namings?

Reply:
I've changed the constant name. The name was a leftover of the previous implementation as StreamIndex (similar to StreamTrees).